src/applications/disability-benefits/686c-674/config/chapters/stepchild-no-longer-part-of-household/stepchild-information/stepchild-information.js

```diff
@@ -47,18 +47,18 @@ export const uiSchema = {
       whoDoesTheStepchildLiveWith: {
         'ui:title': 'Who does this stepchild live with?',
         first: {
-          'ui:title': 'First Name',
+          'ui:title': 'First Name of parent or guardian',
           'ui:required': formData =>
             isChapterFieldRequired(
               formData,
               TASK_KEYS.reportStepchildNotInHousehold,
             ),
         },
         middle: {
-          'ui:title': 'Middle Name',
+          'ui:title': 'Middle Name of parent or guardian',
```

Comment:
Just a heads up, you can add a property called `hideEmptyValueInReview` to `ui:options` to hide this field on the review page - https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vre/definitions/profileAddress.js#L125

It's something that has come up in a11y audits of our other forms, may be worth while to do it here as well.

Reply:
@micahchiang That screenshot was used for two different tickets so the military thing isn't needed for this one.